.github/workflows/main.yml

```diff
@@ -183,7 +183,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.8, 3.11]
+        python-version: [3.8, 3.12]
```

Comment:
I wonder why it is 3.12 and 3.12.0 in some places. Is it something new in the python versioning?

Reply:
From the https://github.com/qiskit-community/qiskit-machine-learning/pull/774#issuecomment-1958091770 above:

> Only the unit tests need to use the exact 3.12.0 version (testtools issue only affects unittests) - the tutorials we again do min and max versions and there its simply 3.12 since it does not need to be down to the exact bug fix version.